src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (Object.prototype.hasOwnProperty.call(invertedRobot, robot[key])) {
```

Comment:
The check for property presence is correct, but it's not necessary to use `Object.prototype.hasOwnProperty.call` here. Instead, you can directly check if the value exists in the `invertedRobot` object using `invertedRobot.hasOwnProperty(robot[key])` or `robot[key] in invertedRobot` which is more concise and easier to read.